uber/templates/preregistration/index.html

```diff
@@ -98,9 +98,9 @@
         <th></th>
     </tr>
 </tfoot>
+{% endif %}
 
 </table></div>
-{% endif %}
 
```

Comment:
this ENDIF here was the actual culprit.  it was placed incorrectly, resulting in the HTML being broken enough that Stripe's code couldn't find its button in the DOM.

basically the ```</table></div>``` here wouldn't be rendered if there was not more than 1 preregistration (which is why it worked if you had 2 or more).  

this also unrelatedly fixes some formatting where the preregs would be shown above or below the disclaimer depending on the # of preregs.

it took hours to figure this out, goddamn : )